Everest03.NET/Controllers/CustomersController.cs

```diff
@@ -0,0 +1,97 @@
+﻿using Everest03.NET.AppServices;
+using Everest03.NET.Validators;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using System;
+using System.Collections.Generic;
+
+namespace Everest03.NET.Controllers
+{
+    [ApiController]
+    [Route("[controller]")]
+    public class CustomersController : ControllerBase
+    {
+        private AppService _service;
+
+        public CustomersController(AppService service)
+        {
+            _service = service ?? throw new ArgumentNullException(nameof(service));
+        }
+
+        [HttpPost(Name = "PostCustomer")]
+        [ProducesResponseType(StatusCodes.Status201Created, Type = typeof(Customer))]
+        [ProducesResponseType(StatusCodes.Status400BadRequest)]
+        public IActionResult Post([FromBody] Customer body)
+        {
+            try
+            {
+                var Id = _service.SetCustomer(body);
+                return new CreatedResult("Post", $"Created customer successfully, ID: {Id}");
```

Comment:
[Quase a mesma coisa](https://github.com/HenriqueSilvaWarren/Everest03.NET/pull/1#discussion_r1063020164) única diferença é que manteria o created apenas retornando o Id

Expor Id do banco também não é uma boa prática PRINCIPALMENTE se for para um sistema externo
```suggestion
                return Created("", Id);
```